docs/howtos/install_epinio_on_rke.md

```diff
@@ -1,35 +1,78 @@
 ---
-sidebar_label: "Installing Epinio On RKE2 (Rancher)"
-sidebar_position: 18
+sidebar_label: "Installing Epinio On RKE2"
+sidebar_position: 19
 title: ""
 ---
 
-# Rancher RKE2 configuration
+# Creating an RKE2 Kubernetes Cluster
+This guide will help you to deploy a suitable RKE2 Kubernetes cluster for Epinio. More details can be found in RKE2 [quickstart](https://docs.rke2.io/install/quickstart/) guide.
 
 This page was written using the following versions:
-* [epinio helm chart 0.7.1](https://github.com/epinio/helm-charts/releases/tag/epinio-0.7.1)
-* RKE2 version v1.22.7+rke2r2
+* [epinio helm chart 1.8.0](https://github.com/epinio/helm-charts/releases/tag/epinio-1.8.0)
+* RKE2 version v1.25.9+rke2r1
+* openSUSE Leap 15.4
 
-## Create an RKE2 cluster
+## Install RKE2 Kubernetes cluster {#install-rke2}
+The following steps are performed using the `root` account on a dedicated machine for the RKE2 server node.
 
-If you don't have an existing cluster, follow the [quickstart](https://docs.rke2.io/install/quickstart/) to create an RKE2 cluster.
+#### 1. Run the installer, start and enable the rke2-server systemd service
+```bash
+curl -sfL https://get.rke2.io | sh -
+systemctl enable --now rke2-server.service
+```
+#### 2. Configure environment variables for operating the RKE2 cluster
+Execute the following commands in the RKE2 node shell and/or store them in the `/root/.bashrc` file (or its equivalent) for persistence.
 
-## RKE2 Prerequisites
+```bash
+export PATH=$PATH:/var/lib/rancher/rke2/bin
+export KUBECONFIG=/etc/rancher/rke2/rke2.yaml
+```
 
-* A load-balancer is mandatory for Epinio to work on RKE2. Here [MetalLB](https://metallb.universe.tf) was used as load-balancer.
-  See [Provision of External IP for LoadBalancer service type in Kubernetes](../howtos/provision_external_ip_for_local_kubernetes) for more information.
+Make sure that you are able to communicate with your new RKE2 cluster by running `kubectl get pods --all`.
+
+## RKE2 cluster prerequisities
+Perform the following steps on your RKE2 node before installing Epinio:
+
+#### 1. Install helm CLI
+```bash
+curl https://raw.githubusercontent.com/helm/helm/main/scripts/get-helm-3 | bash
+```
```

Comment:
As in the other page I don't see this as a "cluster requirement" but more as something that is needed to the operator.

Reply:
I agree but on the other hand it can some time to the operator so I'd keep it there - btw `kubectl` is shipped with RKE2 but `helm` is not.

Reply:
keeping this in place